pillar/nginx/ocw_aka.sls

```diff
@@ -0,0 +1,53 @@
+{% set app_name = 'ocw-aka' %}
+{% set env_settings = salt.cp.get_file_str("salt://environment_settings.yml")|load_yaml %}
+{% set ENVIRONMENT = salt.grains.get('environment', 'ocw') %}
+{% set env_data = env_settings.environments[ENVIRONMENT] %}
+{% set server_domain_names = env_data.purposes[app_name].domains %}
+
+nginx:
+  ng:
+    install_from_ppa: True
+    servers:
+      managed:
+        {{ app_name }}:
+          enabled: True
+          config:
+            - server:
+                - server_name: {{ server_domain_names }}
+                - listen:
+                    - 80
+                - listen:
+                    - '[::]:80'
+                - location /:
+                    - return: 301 https://$host$request_uri
+            - server:
+                - server_name: {{ server_domain_names }}
+                - listen:
+                    - 443
+                    - ssl
+                    - default
+                - listen:
+                    - '[::]:443'
+                    - ssl
+                - root: /ocwdata/apache-tomcat/webapps/
```

Comment:
I don't think we should cargo-cult the tomcat, etc. directories. We should just make it reasonable and semantically relevant for our own sanity. There will be an initial bit of work to update configs, but worth it in the end (I hope...)